src/helpers/issue-fetching.ts

```diff
@@ -296,3 +318,41 @@ function castCommentsToSimplifiedComments(comments: Comments, params: FetchParam
       url: comment.html_url,
     }));
 }
+
+export async function fetchLinkedPrFromIssue(owner: string, repo: string, issueNumber: number, context: Context) {
```

Comment:
Could be better named as it's actually fetching the linked issue from the pull request body